x/emissions/keeper/keeper.go

```diff
@@ -1821,7 +1821,7 @@ func (k Keeper) GetPreviousPercentageRewardToStakedReputers(ctx context.Context)
 
 /// WHITELISTS
 
-func (k *Keeper) IsWhitelistAdmin(ctx context.Context, admin ActorId) (bool, error) {
```

Comment:
surprised this is needed...

Reply:
yeah I don't really understand it fully but it's something about using golang `interface{}` and the reflect package which the `expected_keepers` use. Won't compile otherwise